src/view/pages/FarmerSearch/index.tsx

```diff
@@ -1,107 +1,151 @@
 import * as React from 'react';
 
 import { Content, List, ListItem } from 'native-base';
-import { ScrollView, Text } from 'react-native';
+import { Text, View } from 'react-native';
 import { MapStateToProps, MapDispatchToProps, connect } from 'react-redux';
 
 import createSearchPage, { InjectedSearchProps } from '../../generators/SearchPage';
 import navActions from '../../../store/modules/nav/actions';
 import { Route } from '../../navigation/navigator';
-import createPage from '../../generators/Page/index';
 import { State } from '../../../store/types';
+import { styles } from './style';
 
-/**
- * This is the basic model for the type of farmer object that
- * should be put into array form and given to this page component
- */
-interface farmerList {
+/** This is just a table of phony information to populate the FarmerSearch UI */
+const defaultFarmerList = [{ name: 'Swalleh', phoneNumber: '1-250-234-1234', id: 1 },
+                    { name: 'James', phoneNumber: '1-526-123-8123', id: 2 },
+                    { name: 'Alex', phoneNumber: '1-514-235-6789', id: 3 },
+                    { name: 'Joseph', phoneNumber: '1-922-789-2348', id: 4 },
+                    { name: 'Mary', phoneNumber: '1-626-626-1236', id: 5 },
+                    { name: 'David', phoneNumber: '1-789-231-2345', id: 6 },
+                    { name: 'Michael', phoneNumber: '1-899-781-8786', id: 7 },
+                    { name: 'Mary', phoneNumber: '1-897-768-6780', id: 8 },
+                    { name: 'Peter', phoneNumber: '1-123-564-2315', id: 9 },
+                    { name: 'Jonah', phoneNumber: '1-011-101-1001', id: 10 },
+                    { name: 'Simon', phoneNumber: '1-234-456-7890', id: 11 }];
+
+/** Basic model for the FarmerType object */
+interface FarmerType {
   name: string;
   phoneNumber: string;
   id: number;
 }
```

Comment:
Can you please add  a `// TODO: Move to store` for James when he comes to work on this file w/ redux

Reply:
I think I saw this on your new branch, which also contains comments I left for James. If we could please try to get this branch ready to merge ASAP that would be best for James. Until then @jeinglis I would advise leaving this file alone, and just concentrate on building redux state for farmers, dairy cards, etc. It should probably match the API exposed by the core as closely as possible

Reply:
I see I missed the comment, it's just because of where I put my review comment. I would still wait James before touching this file until his fuzzy search is in, because it changes the file dramatically for you